lib/sensors/sensors.hpp

```diff
@@ -17,6 +17,7 @@ class II2cSensor {
    */
   virtual bool configure()        = 0;
   virtual std::optional<T> read() = 0;
+  virtual ~II2cSensor() {}
```

Comment:
Unsure whether this is in the right file? sensors.hpp is very vague. Just make one i2c.hpp/cpp?

Reply:
`i2c_sensors.hpp`?

Reply:
just so its not confused with the io i2c files